### ISSUE TITLE
feature request: timezone for server time

### ISSUE THREAD

**Issue by yukach** [FoE-Info/FoE-Info-Extension#39]
is it possible to add a timezone to the GbG province time?
for instance. now NY time 9 AM but a lot of players of my guild live in different timezones. I would like to print TZ of server as well. it looks like: 9 AM EST

**Comment by kronikpillow** [comment]
the timezone should be in server time always, so if the server prints it as 9 AM, Info will pick it up as 9 AM, i can possibly hardcode 24h format

**Comment by yukach** [comment]
definitely, the server timezone is the same, Is it possible to detect timezone of the server add add that info about TZ to province time: 9 AM EDT or 21:00 EDT.

**Comment by kronikpillow** [comment]
sorry for the late reply, yes i think it is, i like the idea and will work on it ^^
